src/status_im/utils/ethereum/macros.clj

```diff
@@ -1,13 +1,21 @@
 (ns status-im.utils.ethereum.macros
-  (:require [clojure.string :as string]
-            [clojure.java.io :as io]))
+  (:require [clojure.java.io :as io]))
 
 (defn token-icon-path
   [network symbol]
-  (let [s (str "./resources/images/tokens/" (name network) "/" (name symbol) ".png")]
+  (let [s     (str "./resources/images/tokens/" (name network) "/" (name symbol) ".png")
```

Comment:
isn't it going to make the performance of the manage token asset list even worst?

Reply:
we can load this data right after startup if it will, though we definitely don't need these images on startup